tests/core/test_memory.py

```diff
@@ -293,7 +293,12 @@ def test_empty_model_size(mode):
 
 @pytest.mark.skipif(not torch.cuda.is_available(), reason="Test requires GPU.")
 @pytest.mark.skipif(not _NATIVE_AMP_AVAILABLE, reason="test requires native AMP.")
-@pytest.mark.parametrize('precision', [16, 32])
+@pytest.mark.parametrize(
+    'precision', [
+        pytest.param(16, marks=pytest.mark.skip(reason="no longer valid, because 16 can mean mixed precision")),
```

Comment:
```suggestion
```

Reply:
I suggest not to remove it yet. We need to address this if we want to support the correct number in the summary. 
The author of the PR that added this feature said he/she will do a follow up.